actors/builtin/miner/miner_actor.go

```diff
@@ -1869,17 +1874,21 @@ func (a Actor) ApplyRewards(rt Runtime, params *builtin.ApplyRewardParams) *abi.
 
 		newlyVested, err := st.AddLockedFunds(store, rt.CurrEpoch(), rewardToLock, lockedRewardVestingSpec)
 		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to lock funds in vesting table")
+		/// XXX am unsure if this is correct! should i be logging amountUnlocked, or VestingSum from addLockedFunds?? could the return value be a bug? confused about what's going on.
```

Comment:
I think the return value is correct -- As a side effect of adding locked funds,  some vested funds could be unlocked. The return is just communicating this update.

I don't think this is worth logging at all.